specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -4349,6 +4356,20 @@
           },
           {
             "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation.",
+            "enum": [
+              "userData"
+            ],
+            "x-ms-enum": {
+              "name": "ExpandTypes",
+              "modelAsString": false
```

Comment:
If you plan to add values to this enum in the future and if it can be null, I suggest setting modelAsString to 'true'.

Reply:
changed to true ty